apps/vercel/frontend/src/components/config-screen/ContentTypePreviewPathSection/InformationalModal/InformationalModal.tsx

```diff
@@ -16,58 +17,62 @@ export const InformationalModal = ({ onClose, isShown }: Props) => {
     copies.configPage.contentTypePreviewPathSection.exampleModal;
 
   return (
-    <Modal onClose={onClose} isShown={isShown} size="large">
-      {() => (
-        <>
-          <Modal.Header title={title} onClose={onClose} />
-          <Modal.Content>
-            <Flex flexDirection="column" alignItems="baseline" gap={tokens.spacingS}>
-              <Flex flexDirection="column" gap={tokens.spacingL}>
-                <Box>
-                  <Paragraph className={styles.firstParagraph}>{exampleOne.description}</Paragraph>
-                  <GrayInfoBox>{exampleOne.example}</GrayInfoBox>
-                </Box>
+    <SDKProvider>
```

Comment:
just needed to add the SDKProvider to the modal (since this is a sibling node to the app itself when rendered).